models/content_test.go

```diff
@@ -0,0 +1,125 @@
+package models_test
+
+import (
+	"testing"
+
+	"github.com/ONSdigital/dp-topic-api/apierrors"
+	"github.com/ONSdigital/dp-topic-api/models"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestContentValidation(t *testing.T) {
+	Convey("Given an empty content, it is successfully validated", t, func() {
+		content := models.Content{
+			State: models.StateCreated.String(), //!!! remove topic out of the enum name
```

Comment:
I think this comment has been addressed and can be removed

Reply:
done in later code that is waiting on this PR before i can do its PR.